src/features/counterfactual/hooks/useDeployGasLimit.ts

```diff
@@ -92,6 +92,7 @@ export const estimateBatchDeploymentTransaction = async (
   // 2. Add a simulate call to the predicted SafeProxy as second transaction
   const transactionDataToEstimate: string = simulateTxAccessorContract.encode('simulate', [
     safeTransaction.data.to,
+    // @ts-ignore
     safeTransaction.data.value,
```

Comment:
It complains that safeTransaction.data.value needs to be bigint instead of string. I am not sure why this change is suddenly necessary or why it worked before. We could try changing it and see if it still works.

Reply:
Tested this with calling `BigInt(safeTransaction.data.value)` which also works so I will remove the @ts-ignore